.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,39 @@
+name: Feature Request
+description: Suggest an idea for the COBOL Programming Course
+labels: ['enhancement']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request. Please fill up the information below and we will get back to you as soon as possible.
```

Comment:
Please fill in the information...

Also, do we want to recommend searching the issues to see if the enhancement exists like you did for the Bug Report template?

Reply:
Updated to include the recommendation